index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="assets/style/style.css">
+  <title>Cinema | Showcase</title>
+</head>
+<body>
+  <header class="header-bar">
+    <nav class="nav-bar">
+      <img class="humberger" src="assets/icons/menu.png" alt="menu" id="menubtn">
```

Comment:
- [ ] Kindly add the `social icons` missing on your `navigation bar`  to match the original design

![image](https://user-images.githubusercontent.com/98453979/225987515-10180fe6-5729-4e49-b42f-f1401e5032cf.png)

Reply:
Done